README.md

```diff
@@ -4,35 +4,90 @@ NodeJS >=18.19.0
+│   │   ├── 404.svelte
+│   ├── scss // Styles that are shared across multiple components
+│   │   └── variables.scss // Defines a preset of colors and values to be used uniformly across the project - need a color? Import from here!
+│   └── types.ts // A central place for types - if types are specific for a component (or similar) and not shared across the project, put them in the respective component file
+├── static // These files will be mounted on the root. For example, ./static/example.jpg will later be accessible under https://bsaber.com/example.jpg
+│   ├── admin
+│   │   ├── config.yml // This files defines the config and data model for Netlify CMS - if you want to edit, add or remove a collection some changes will happen here
+│   │   └── index.html // Entry point for the Netlify CMS page for admins
+│   ├── characteristics // Folder containing the images for the diffs on the map cards
+│   ├── favicon.png // This file is the favicon of the website
+│   ├── uploads // Directory, that contains uploaded images from netlify CMS
+├── svelte.config.js // Svelte(kit) configuration - contains the configurations for crawling and SSG as well
+├── update-cover-url-overwrites.sh // See "Utility Scripts" for info
+```
+
+## Project Routing
+
+The routes are determined by file path starting at `src/routes`. For example, (`src/routes/community-hub/+page.svelte`)[src/routes/community-hub/+page.svelte] renders under (https://bsaber.com/community-hub)[https://bsaber.com/community-hub] or in the local development environment under (http://localhost:3000/community-hub)[http://localhost:3000/community-hub].
+
+Note the following when it comes to those:
+
+1. **We Statically Generate our Site**: You cannot rely on server side routing to dynamically update the pages. These only execute once when rendering and generating the static HTML and CSS files and cannot react to, for example, a change in the file system. Creating and managing files will thus also not usually make sense.
+
+2. **Pages with Path Parameters Need to be Linked**: Paths that contain path parameters such as `[page]` need to be linked to somehwere from another page that either has no path parameter or is linked by a page fulfilling these conditions. Basically, they need to be crawlable by following links. If a page cannot be crawled starting from any non-parameter path it will simply exist in the final product.
+
+3. **`npm run dev` =/= Production Environment**: You cannot rely on it 100%. For example, non-rendered pages will still work with `npm run dev` since it actually executes server-side.
+
+4. **You usually want to stick to `+page.server.ts` not `+page.ts`**: The difference between the two is, that the `.server.ts` only executes server side (so once when rendering). The `+page.ts` will run server and client side. Due to the way we use server-side code it usually doesn't make sense to use `+page.ts` and can quickly result in confusing errors.
```

Comment:
The comma should be placed before "is".

Reply:
I would say there should be no comma here at all